src/else.js

```diff
@@ -11,6 +11,16 @@ export class Else extends IfCore {
     this._registerInIf();
   }
 
+  bind(bindingContext, overrideContext) {
+    super.bind(bindingContext, overrideContext);
+
+    if (!this.ifVm) {
```

Comment:
@jods4 I was thinking about if the else is specified before the if so that it could be in the context but bound before. But after you said it, I recognized that this might not be possible since the if is registered in the constructor.